README.md

```diff
@@ -10,16 +10,12 @@ A test harness for the IDAM-Express-Middleware
 
 ## Installation
 
-As of now, this module is published only in a private repository.
-We are working on publishing this project to NPM.
-Until then, the package can be installed from its github URL, examples:
-
 ```bash
 # Install the latest version
-yarn add https://github.com/hmcts/div-idam-test-harness
+yarn add "@hmcts/div-idam-test-harness"
 
 # Install a specific version
-yarn add https://github.com/hmcts/div-idam-test-harness#1.0.11
+yarn add "@hmcts/div-idam-test-harness@1.2.1"
```

Comment:
Should this be 1.3.0?

Reply:
I wanted to give an example with a version that wasn't the latest. Otherwise, there's no visible change in the outcome.